app/models/grda_warehouse/client_file.rb

```diff
@@ -50,7 +50,18 @@ class ClientFile < GrdaWarehouse::File
         ).or(arel_table[:user_id].eq(user.id))
 
         sql = sql.or(arel_table[:id].in(Arel.sql(consent_forms.select(:id).to_sql))) if GrdaWarehouse::Config.get(:consent_visible_to_all)
-        sql = sql.or(arel_table[:id].in(Arel.sql(verified_homeless_history.select(:id).to_sql))) if can_view_homeless_verification_pdfs
+
+        # Include hoomeless verification PDfs based on site config:
+        # If using 'release' method, show all files ONLY if there is a valid release. If not, only show your own files.
+        # If using any other method, show all files.
+        if can_view_homeless_verification_pdfs
```

Comment:
If this is hitting the normal permission system, it should have the ?
```suggestion
        if can_view_homeless_verification_pdfs?
```

Reply:
It's a variable set on line 41 above:
```
can_view_homeless_verification_pdfs = GrdaWarehouse::Config.get(:verified_homeless_history_visible_to_all) || user.can_generate_homeless_verification_pdfs?
```

Do we want to continue to use that logic?

Reply:
that's good, just wasn't in the diff, so I missed it.